adcircpy/forcing/winds/best_track.py

```diff
@@ -94,6 +113,30 @@ def __init__(
 
         super().__init__(nws=nws, interval_seconds=interval_seconds)
 
+    def summary(self, output: Union[str, os.PathLike] = None):
```

Comment:
Now it's better. You either do `print(bt.summary())` or `bt.summary('stats.txt')` or both at the same time: `print(bt.summary('stats.txt'))`

Reply:
@krober10nd  Let me know if this makes sense to you, or if you have a different vision.

Reply:
yep, I agree with this behavior. In the way it is now, the summary can be written to a file or printed to the screen. The user can also manipulate it how they please for further processing.